sdk/communication/communication-sms/samples/v1/javascript/README.md

```diff
@@ -51,7 +51,7 @@ node sendSms.js
 Alternatively, run a single sample with the correct environment variables set (setting up the `.env` file is not required if you do this), for example (cross-platform):
 
 ```bash
-npx cross-env COMMUNICATION_CONNECTION_STRING="<communication connection string>" node sendSms.js
+npx cross-env COMMUNICATION_SAMPLES_CONNECTION_STRING="<communication samples connection string>" FROM_PHONE_NUMBER="<from phone number>" AZURE_PHONE_NUMBER="<azure phone number>" TO_PHONE_NUMBERS="<to phone numbers>" AZURE_PHONE_NUMBER="<azure phone number>" node sendSms.js
```

Comment:
`AZURE_PHONE_NUMBER="<azure phone number>" node sendSms.js`
I think these instructions are a little unclear. Cause technically if we have `TO_PHONE_NUMBERS` and `FROM_PHONE_NUMBERS` then setting the `AZURE_PHONE_NUMBER` become moot. We should strive for a bit more clarity.

Reply:
this is auto generated

Reply:
We want to keep `AZURE_PHONE_NUMBER` cause that is the one at the moment that the pipeline is injecting. I can make a new property in the config for the new properties `TO_PHONE_NUMBERS` and `FROM_PHONE_NUMBERS` but for the time being this should be fine